react-frontend/src/views/DoctorSignIn.js

```diff
@@ -42,16 +43,17 @@ function DoctorSignIn() {
     })
     const resData = await res.json()
     console.log(resData)
-    localStorage.setItem('doctorClinicAppToken', resData.token)
-    history.push('/home')
+    localStorage.setItem('doctorClinicAppData', JSON.stringify(resData))
```

Comment:
You should always wrap localStorage usage in a try catch, otherwise it can throw exceptions and take down the UI thread; app crashes, bad experience.

Reply:
Okay